rspec/solver_spec.rb

```diff
@@ -0,0 +1,29 @@
+require '../solver'
```

Comment:
Please use `require_relative` instead of `require` because `require` is used to load gems in Ruby and `require_relative` is used to load local files and directories. There is an error which is fixed when i use `require_relative`.

![error](https://user-images.githubusercontent.com/35504149/203846144-a9227017-4a15-4b4d-8fc8-d60f3609d0c1.png)